WooCommerce/Classes/POS/Presentation/ParentProductCardView.swift

```diff
@@ -1,34 +1,37 @@
 import SwiftUI
-import struct Yosemite.POSParentProduct
 
 /// Displays a card for a parent product in POS.
-struct ParentProductCardView: View {
-    private let parentProduct: POSParentProduct
+struct ParentProductCardView<DetailView: View>: View {
+    private let name: String
+    private let imageSource: String?
+    private let detailView: DetailView
```

Comment:
If you keep this – I think you can make this a ViewBuilder, so you only call it when it's used not at init time.